.github/workflows/box64-upload.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Upload box64 source RPM
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'box64/**'
+  workflow_dispatch:
+
+env:
+  PACKAGE_NAME: box64
+
+jobs:
+  upload:
+    name: Upload box64 source RPM to Fedora Copr
+    runs-on: ubuntu-22.04
+    container:
+      image: fedora:41
+    env:
+      FEDORA_COPR_LOGIN: ${{ secrets.FEDORA_COPR_LOGIN }}
+      FEDORA_COPR_TOKEN: ${{ secrets.FEDORA_COPR_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install RPM build dependencies
+        run: dnf install -y copr-cli rpm-build rpmdevtools
+      - name: Create the necessary RPM build directories
+        run: mkdir -p ~/rpmbuild/SOURCES/
+      - name: Copy internal source files
+        run: cp ${GITHUB_WORKSPACE}/${PACKAGE_NAME}/* ~/rpmbuild/SOURCES/
+      - name: Download external source files
+        run: spectool -g -R ${GITHUB_WORKSPACE}/${PACKAGE_NAME}/*.spec
+      - name: Build the source RPM
+        run: rpmbuild -bs ${GITHUB_WORKSPACE}/${PACKAGE_NAME}/*.spec
+      - name: Create the local configuration directory
+        run: mkdir ~/.config/
+      - name: Create the copr-cli configuration file
+        run: |
+          cat <<EOF > ~/.config/copr
+          [copr-cli]
+          login = ${FEDORA_COPR_LOGIN}
+          username = playtron
+          token = ${FEDORA_COPR_TOKEN}
+          copr_url = https://copr.fedorainfracloud.org
+          EOF
+      - name: Upload the source RPM to Fedora Copr
+        run: copr-cli build gaming ~/rpmbuild/SRPMS/${PACKAGE_NAME}-*.src.rpm
```

Comment:
This needs to exclude the x86 32-bit architecture RPM build.

`--exclude fedora41-i386 --exclude fedora42-i386`

Similar to: https://github.com/playtron-os/rpm-specs-gaming/blob/9cc4a3d56a76f556b459e355a7d4d91abe1462d9/.github/workflows/valve-firmware-upload.yml#L43

We can technically keep the x86_64 architecture but that build of Box64 is too slow and not useful right now.

`--exclude fedora-41-x86_64 --exclude  fedora-42-x86_64`

Reply:
Should it be `--exclude` or `--exclude-chroot`?